dev/reformat

```diff
@@ -9,7 +9,7 @@ if ! command -v ruff &> /dev/null; then
 fi
 
 # run ruff linter
-ruff check --fix ./api
+ruff check --fix --unsafe-fixes ./api
```

Comment:
Let's keep this line as it is. Removing `--unsafe-fixes` will allow poeple to check it manually, which might be better.

```suggestion
ruff check --fix ./api
```

Reply:
Alright. SGTM.